Tests/scripts/infrastructure_tests/common_test.py

```diff
@@ -1,147 +1,174 @@
-from Tests.scripts import common
+from Tests.scripts.common import get_reviewer, get_person_in_charge, are_pipelines_in_order, is_pivot, get_slack_user_name
+import pytest
+from requests_mock import MockerCore
 
 
-def test_person_in_charge(mocker):
+def test_get_person_in_charge(mocker):
     """
     Given a commit object
-    When person_in_charge is called on it
-    Then it should return the expected name, email and PR link
+    When get_person_in_charge is called on it
+    Then it should return the expected name and PR link
     """
     commit = mocker.Mock()
     commit.author_name = 'John Doe'
     commit.author_email = 'john@doe.com'
     commit.title = 'Merge branch \'master\' into branch-name (#123)'
 
-    expected = ('John Doe', 'john@doe.com', 'https://github.com/demisto/content/pull/123')
+    expected = ('John Doe', 'https://github.com/demisto/content/pull/123')
 
-    result = common.person_in_charge(commit)
+    result = get_person_in_charge(commit)
 
     assert result == expected
 
 
-def test_are_pipelines_in_order_as_commits_true(mocker):
+@pytest.mark.parametrize(('pipeline1_date, pipeline2_date, expected'), (
+    pytest.param('2020-01-01T00:00:00Z', '2020-01-02T00:00:00Z', False, id="pipelines not in order"),
+    pytest.param('2020-01-02T00:00:00Z', '2020-01-01T00:00:00Z', True, id="pipelines in order")))
+def test_are_pipelines_in_order(mocker, pipeline1_date, pipeline2_date, expected):
     """
-    Given a list of commits and two pipeline SHAs
-    When the pipeline SHAs are in the same order as the commits
-    Then it should return True and the commit that triggered the later pipeline
-
+    Given:
+        - Two pipelines with their created_at dates.
+    When:
+        - are_pipelines_in_order is called on them.
+    Then:
+        - It should return the expected result.
+          scenario 1: pipeline1.created_at > pipeline2.created_at -> False
+          scenario 2: pipeline1.created_at < pipeline2.created_at -> True
```

Comment:
please refer to https://www.businessanalysisexperts.com/gherkin-user-stories-given-when-then-examples/ 
for writing Given-When-Then